Gemfile

```diff
@@ -15,13 +15,15 @@ end
 group :test do
   gem 'activesupport'
   gem 'erubis'
+  gem 'faraday-typhoeus'
```

Comment:
Is this becoming a gem dependency? Does it belong in .gemspec?

Reply:
This was added as testing dependency because it was required for the 'non default adapter' spec in `spec/slack/web/client_spec.rb:177` and the adapters are no longer bundled in.